android/app/src/main/AndroidManifest.xml

```diff
@@ -32,9 +32,6 @@
     <uses-permission android:name="android.permission.READ_CONTACTS" />
     <uses-permission android:name="android.permission.READ_CALENDAR" />
     <uses-permission android:name="android.permission.WRITE_CALENDAR" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
`android.permission.READ_EXTERNAL_STORAGE` and `android.permission.WRITE_EXTERNAL_STORAGE` should still stay. In order to allow device to store data on and get data from external storage (in case internal storage is full)

Reply:
done, thanks